PIPS/pip-008.md

```diff
@@ -0,0 +1,28 @@
+    PIP: 008
+    Title: On-A-Budget mode (aka Doshirak mode)
+    Author: <Kosta Korenkov>
+    Created: <2018-03-18>
+    Affects: PIP004, PIP005, PIP007
+    Status: draft
+
+
+## Simple Summary
+Cut our costs to bare minimum while we are looking for extra funding. This PIP is basically a bunch of amendments to PIP004, PIP005, PIP007 which are activated conditionally.
+
+## Motivation
+We are running out of cash and our top priority should be to look for investors/grants and that's what we should spent most of the money for.
+
+## Activation/Deactivation
+This PIP comes in effect on Monday 00:00 UTC if Escrow council contains less than 50,000 DAI worth of ETH.
```

Comment:
i would make this noon, so we can better look up the current rate of ETH/DAI

Reply:
It would be hard to coordinate with PIP003 — people may claim basic income before noon and we will discriminate those who claimed after noon

Reply:
ok, just saying the condition depends on the ETH price, and where would we get the historic data?